javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -803,6 +816,26 @@ protected void generateLongRunningBeginAsync(ClientMethod clientMethod, JavaType
         });
     }
 
+    /**
+     * Generate long running begin async method for protocol client
+     * @param clientMethod client method
+     * @param typeBlock type block
+     */
+    protected void generateProtocolLongRunningBeginAsync(ClientMethod clientMethod, JavaType typeBlock) {
+        String contextParam = "requestOptions != null ? requestOptions.getContext() : Context.NONE";
+        String httpPipelineParam = clientMethod.getClientReference() + ".getHttpPipeline()";
+        String pollingStrategy = "new DefaultPollingStrategy<>(" + httpPipelineParam + ", null, " + contextParam + ")";
+        typeBlock.annotation("ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)");
+        writeMethod(typeBlock, clientMethod.getMethodVisibility(), clientMethod.getDeclaration(), function -> {
+            function.line("return PollerFlux.create(Duration.ofSeconds(%s),", clientMethod.getMethodPollingDetails().getPollIntervalInSeconds());
+            function.increaseIndent();
+            function.line("() -> this.%s(%s),", clientMethod.getProxyMethod().getSimpleAsyncRestResponseMethodName(), clientMethod.getArgumentList());
+            function.line(pollingStrategy + ",");
+            TemplateUtil.writeLongRunningOperationTypeReference(function, clientMethod);
+            function.decreaseIndent();
+        });
+    }
```

Comment:
Does it need to be similar to `generateLongRunningBeginAsync` method? As there is `polling` options that can allow dev to supply different `PollingStrategy` other than the `DefaultPollingStrategy`.